src/com/ss/ita/kata/implementation/Nastia3/SevenImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.ss.ita.kata.implementation.Nastia3;
+
+import com.ss.ita.kata.Seven;
+import java.text.DecimalFormat;
+
+public class SevenImpl  implements Seven {
+    @Override
+    public long newAvg(double[] arr, double navg) {
+        return 0;
+    }
+
+    @Override
+    public String seriesSum(int n) {
+        DecimalFormat df = new DecimalFormat("0.00");
+        if(n>=1){
+            double sum = 1.0;
+            for(int i =1;i<n;i++){
+                sum += 1.0/((3*i+1)*1.0);
```

Comment:
This multiplying by 1 is redundant. Am I wrong?

Reply:
fixed